tests/GenerateDocumentationTest.php

```diff
@@ -114,6 +114,18 @@ public function can_skip_single_routes()
         $this->assertContains('Processed route: [GET] api/test', $output);
     }
 
+    /** @test */
+    public function can_skip_non_existent_response_files()
+    {
+        RouteFacade::get('/api/non-existent', TestController::class.'@withNonExistentResponseFile');
+
+        config(['apidoc.routes.0.match.prefixes' => ['api/*']]);
+        $output = $this->artisan('apidoc:generate');
+
+        $this->assertContains('Skipping route: [GET] api/non-existent', $output);
```

Comment:
Should we necessarily skip this, though? I'm wnodering how big of an issue a missing response file isenough to completely write off a route? Perhaps not. We can simply emit the warning and return no response from that strategy.

Reply:
I think we'd need to implement try/catch a bit earlier in that case, and we'd need to pass IO methods to the Generator - you happy with that?

Reply:
Hmm. Okay, let's just go along with this. Anyone who doesn't like it can always open an issue.